apps/teams-test-app/src/components/AuthenticationAPIs.tsx

```diff
@@ -128,6 +128,38 @@ const Authenticate = (): React.ReactElement =>
     },
   });
 
+const externalAuthParams = {
+  url:
+    'https://teams-test-tab.azurewebsites.net/auth_start.html?authId=b9237908-2150-497a-8e47-8bf3ebf81b77&oauthRedirectMethod={oauthRedirectMethod}',
+  isExternal: true,
+};
+
+const AuthenticateExternal = (): React.ReactElement =>
+  ApiWithoutInput({
+    name: 'authentication.authenticate3',
```

Comment:
can this be `authentication.authenticate.external`?

Reply:
done